### PR TITLE
Simplify and generalize health check

### PR DESCRIPTION
**Pull request by yakutovicha**
This commit adds a startup script that runs at the very last step. Its
job is to create file /root/INIT_COMPLETED signalling that all init scripts
are completed.

### CODE REVIEW

**Review by ltalirz — CHANGES_REQUESTED** [review]
Thanks @yakutovicha - after having a look at what [HEALTHCHECK actually does](https://docs.docker.com/engine/reference/builder/#healthcheck), I'm not convinced that this solution is ideal - this will cause docker to check every 10s the status of the container... forever... while the result of the check will always stay the same (after initialization).

I would say, for the aiida-prerequisites container, the right thing to check is clearly that both rabbitmq and postgres are up and running.

Is this possible? Also, what was your reason for moving away from checking the postgres status?

P.S. the default timeout for health checks is 30s, while postgres and rabbitmq should respond much quicker. We can probably put `--timeout 5s`

**Comment by yakutovicha** [comment]
so as discussed offline, here I made a different solution: `wait-for-services` script that waits until all startup scripts are finished. Inspired by https://github.com/phusion/baseimage-docker/issues/44 and https://github.com/phusion/baseimage-docker/pull/192

File: bin/wait-for-services [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -em
+
+while true; do
```
Comment (ltalirz) [inline bin/wait-for-services:4]: should there not be some sort of wait in between?
how often will this check?

Comment (yakutovicha) [reply bin/wait-for-services:4]: indeed, it should sleep a bit.

**Review by ltalirz — APPROVED** [review]
thanks @yakutovicha , looks good!

**Pull request merged**